src/main/java/org/ozwillo/dcexporter/model/ui/AuditLogWapper.java

```diff
@@ -21,13 +21,22 @@
 
     @JsonProperty
     private String datasetUrl;
+    @JsonProperty
+    private String orgaName;
```

Comment:
- Please prefer the use of consistent names : organizationName instead of orgaName. We have auto-completion features nowadays ;)